<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDoctor</class>
 <widget class="QDialog" name="AddDoctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 117, 143);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatid">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatname">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linedocname">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="linedocid">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbldocdetails">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Doctor Details</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbllenspre1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Lens Preset One</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(4, 54, 22);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(4, 54, 22);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbllenspre2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Lens Preset Two</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(4, 54, 22);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbllenspre3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Lens Preset Three</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>460</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(4, 54, 22);
font: 75 27pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);
font:bold;</string>
   </property>
   <item>
    <property name="text">
     <string>SRKT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SRKTII</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HOFFER Q</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HOLLADAY</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HAIGIS</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbllensformula">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Primary Formula</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ButDocSave">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>560</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButDocBack">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>560</y>
     <width>181</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
